### PR TITLE
Changed file names in folder Patterns

### PR DESCRIPTION
**Pull request by AnjumanHasan**

### CODE REVIEW

**Comment by tushar5526** [comment]
Rename the folder to `FunctionAndArrays`  instead of Function*s*

**Comment by AnjumanHasan** [comment]
I have already named the folder as FunctionAndArrays

**Comment by tushar5526** [comment]
It is Functions instead of `Function` remove the extra **s**

File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/FunctionsAndArrays/DigitFrequency.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#include<iostream>
```
Comment (tushar5526) [inline Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/FunctionsAndArrays/DigitFrequency.cpp:1]: Should be FunctionAndArrays

Comment (AnjumanHasan) [reply Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/FunctionsAndArrays/DigitFrequency.cpp:1]: okay sir.

**Comment by AnjumanHasan** [comment]
Sir i,ve deleted the folder and added a new one

**Comment by AnjumanHasan** [comment]
I've resolved the renaming issues and added files,

**Pull request merged**